### PR TITLE
Major Refactoring of Code to Vue.js

### PR DESCRIPTION
**Pull request by avishj**

### CODE REVIEW

**Comment by vanshg395** [comment]
while refactoring u removed app code from branch

either add it back or push ur code to a separate branch

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+
+# Editor directories and files
+.idea
+.vscode
```
Comment (vinitshahdeo) [inline .gitignore:17]: @avishj I see `.vscode` here but still I can see the `.vscode` directory in the repo. Please delete the `.vscode folder from the repo. Ideally we shouldn't push the local editor settings.

**Comment by avishj** [comment]
@vanshg395 
The app was never on my repo, all my PR's have been without the app code. I don't think that the app code would get deleted if you pull my request. Correct me if I'm wrong.

File: .gitignore (hunk 1 above)
Comment (avishj) [reply .gitignore:17]: Ok, missed that one, will fix it

Comment (avishj) [reply .gitignore:17]: @vinitshahdeo I had a look at the repo through File Explorer, couldn't find the .vscode directory, could you please tell me the location of that directory?

Comment (vinitshahdeo) [reply .gitignore:17]: `.vscode` directory remain hidden. Check for hidden files and folders.

Comment (akshatvg) [reply .gitignore:17]: @avishj you can find .vscode on your code editor like VS Code too.

**Comment by vanshg395** [comment]
App was never on your repo because you never rebased you repo.

**Pull request closed without merging**